gsms/gsms.py

```diff
@@ -486,25 +484,29 @@ def main() -> None:
                 print(f'working-dir: {working_dir}')
                 print(f'path: {cmd}')
 
-                src_image = os.path.join(
-                    nvidia_images_base_dir,
-                    short_name,
-                    gfe_apps["metadata"][short_name]["c"],
-                    f"{short_name}-box-art.png"
-                )
-                dst_image = os.path.join(args.image_path, f'{short_name}.png')
-
-                if not args.dry_run:
-                    copy_image(src_image=src_image, dst_image=dst_image)
-
-                add_game(
-                    sunshine_apps=sunshine_apps,
-                    name=name,
-                    logfile=f"{short_name}.log",
-                    cmd=cmd,
-                    working_dir=working_dir,
-                    image_path=dst_image
-                )
+                if short_name in gfe_apps["metadata"]:
+                    src_image = os.path.join(
+                        nvidia_images_base_dir,
+                        short_name,
+                        gfe_apps["metadata"][short_name]["c"],
+                        f"{short_name}-box-art.png"
+                    )
+                    dst_image = os.path.join(args.image_path, f'{short_name}.png')
+
+                    if not args.dry_run:
+                        copy_image(src_image=src_image, dst_image=dst_image)
+
+                    add_game(
+                        sunshine_apps=sunshine_apps,
+                        name=name,
+                        logfile=f"{short_name}.log",
+                        cmd=cmd,
+                        working_dir=working_dir,
+                        image_path=dst_image
+                    )
+
+                    # Increase count here to exclude some stuff
+                    count += 1
```

Comment:
```suggestion
                if short_name not in gfe_apps["metadata"]:
                    continue

                src_image = os.path.join(
                    nvidia_images_base_dir,
                    short_name,
                    gfe_apps["metadata"][short_name]["c"],
                    f"{short_name}-box-art.png"
                )
                dst_image = os.path.join(args.image_path, f'{short_name}.png')

                if not args.dry_run:
                    copy_image(src_image=src_image, dst_image=dst_image)

                add_game(
                    sunshine_apps=sunshine_apps,
                    name=name,
                    logfile=f"{short_name}.log",
                    cmd=cmd,
                    working_dir=working_dir,
                    image_path=dst_image
                )

                count += 1
```

Reply:
I do like this version more as it relies on the early continue up above and thus means less nesting.